classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWBT-REQ-0053:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Manually create QG with execution butler.
+    - Run butler.get against execution butler.
+  notes:
+    - Execution butler may be removed in favor of something better (prototyping on DM-32072) by the time we run this test, but whatever replaces it will also satisfy this requirement.
+DMS-MWBT-REQ-0054:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Manually create QG with execution butler.
+    - Run butler.get against execution butler for ref that does not exist.
+  notes:
+    - Execution butler may be removed in favor of something better (prototyping on DM-32072) by the time we run this test, but whatever replaces it will also satisfy this requirement.
+DMS-MWBT-REQ-0055:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Enable datastore caching in a Butler client in RSP (or any S3-backed repo).
+    - Run butler.get twice, check (e.g. trace logs) that the second comes from cache.
+DMS-MWBT-REQ-0056:
+  status: unclear
+  priority: 1a
+  notes:
+    Use cases like the one that motivated this requirement assumed a butler that mediated a lot of database access.  Since then we've move toward approaches that us parquet instead, and it's not clear that butler should be playing a role in those workflows.
+DMS-MWBT-REQ-0057:
+  status: unclear
+  priority: 1a
+  notes:
+    See DMS-MWBT-REQ-0056
+DMS-MWBT-REQ-0058:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Enable datastore caching in a Butler client in RSP (or any S3-backed repo).
+    - Run butler.get twice, check (e.g. trace logs) that the second comes from cache.
+  reuse_script: DMS-MWBT-REQ-0055
+  notes:
+    This doesn't really look distinct from DMS-MWBT-REQ-0055 anymore; I think 0055 was perhaps supposed to be some kind of shared-filesystem proxy for something that lives on even slower storage, like tape.  But the specs aren't different enough for me to propose different tests for them.
+DMS-MWBT-REQ-0036:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Run a `PipelineTask` against a local SQLite+POSIX repo.
+    - Run the same `PipelineTask` against a PostgreSQL+POSIX repo.
+    - Run the same `PipelineTask` against a PostgreSQL+S3 repo.
+  notes:
+    The "same interface" demanded by this requirement was really a core design constraint, so now that we have a butler at all it's trivially verifiable.
+DMS-MWBT-REQ-0037:
+  status: feature-complete
+  priority: 1a
+  script:
+    - Instantiate a butler on RSP targeting DP0.x collections.
+    - Call Butler.get.
+DMS-MWBT-REQ-0038:
+  status: unclear
+  priority: 1a
+  notes:
+    Lots of assumptions here that I'm not sure are valid anymore, involving VOSpace and notebook-launched batch jobs.  Probably needs to be rewritten to reflect (or ideally not depend at all) on current designs.
```

Comment:
I want to hear from @gpdf about how much of this is still relevant before developing any opinion of my own on what to do with this requirement.

Reply:
I think we could at least drop the priority of this to P1b - I don't see this as necessecary for start of commissioning